modules/microshift-migrating-ipv6-dual-stack-config.adoc

```diff
@@ -0,0 +1,161 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-migrating-ipv6-dual-stack-config_{context}"]
+= Migrating a {microshift-short} to IPv6 dual-stack networking
+
+You can convert your IPv4 single-stack cluster to dual-stack cluster networking that supports IPv4 and IPv6 address families by setting two entries in the service and cluster network parameters in the {microshift-short} configuration file.
+
+* The first IP family in the configuration is the primary IP stack in the cluster.
+* After the cluster is migrated to dual-stack networking and has restarted, enable pods and services for dual-stack networking by restarting them.
```

Comment:
Maybe worth mentioning that this is only required for user's workloads, system ones will do automatically by MicroShift.